### ISSUE TITLE
Error message on dedicated server

### ISSUE THREAD

**Issue by PvtFudgepants** [ArsenicPurple/BossHealthMod#4]
I'm trying to use this mod on my dedicated server but I'm getting this error message whenever someone tries to open a loot bag.

`java.lang.IllegalArgumentException: Missing required parameters: [<parameter minecraft:damage_source>, <parameter minecraft:origin>]
	at net.minecraft.world.level.storage.loot.LootParams$Builder.m_287235_(LootParams.java:132) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at co.basin.betterbosses.item.LootBagItem.createLoot(LootBagItem.java:66) ~[multiplayerbosses-1.1.2.jar%23916!/:1.1.2]
	at co.basin.betterbosses.item.LootBagItem.m_7203_(LootBagItem.java:49) ~[multiplayerbosses-1.1.2.jar%23916!/:1.1.2]
	at net.minecraft.world.item.ItemStack.m_41682_(ItemStack.java:275) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.server.level.ServerPlayerGameMode.m_6261_(ServerPlayerGameMode.java:286) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.m_5760_(ServerGamePacketListenerImpl.java:1087) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.network.protocol.game.ServerboundUseItemPacket.m_5797_(ServerboundUseItemPacket.java:29) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.network.protocol.game.ServerboundUseItemPacket.m_5797_(ServerboundUseItemPacket.java:7) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.network.protocol.PacketUtils.m_263899_(PacketUtils.java:22) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.server.TickTask.run(TickTask.java:18) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:156) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:770) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:161) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:130) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.server.MinecraftServer.m_129961_(MinecraftServer.java:753) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.server.MinecraftServer.m_7245_(MinecraftServer.java:747) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.m_18699_(BlockableEventLoop.java:115) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.server.MinecraftServer.m_130012_(MinecraftServer.java:732) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:665) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[server-1.20.1-20230612.114412-srg.jar%231105!/:?]
	at java.lang.Thread.run(Thread.java:840) ~[?:?]`

Seems like this is happening with any modded boss loot bag. Wither and ender dragon work fine.

**Comment by ACE1337x** [comment]
im running into this issue too, but my problem is happening with the ender dragon loot bag. im using sinytra connector and my dragon is supposed to drop loot from the rpg series mods

```
[23:40:30] [Server thread/ERROR] [minecraft/PacketUtils]: Failed to handle packet net.minecraft.network.protocol.game.ServerboundUseItemPacket@774a751, suppressing error
java.lang.IllegalArgumentException: Missing required parameters: [<parameter minecraft:origin>, <parameter minecraft:damage_source>]
        at net.minecraft.world.level.storage.loot.LootParams$Builder.m_287235_(LootParams.java:132) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:mixin,pl:connector_pre_launch:A,re:classloading,pl:connector_pre_launch:A}
        at co.basin.betterbosses.item.LootBagItem.createLoot(LootBagItem.java:66) ~[multiplayerbosses-1.1.2.jar%23489!/:1.1.2] {re:classloading}
        at co.basin.betterbosses.item.LootBagItem.m_7203_(LootBagItem.java:49) ~[multiplayerbosses-1.1.2.jar%23489!/:1.1.2] {re:classloading}
        at net.minecraft.world.item.ItemStack.m_41682_(ItemStack.java:275) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:mixin,xf:fml:forge:itemstack,pl:connector_pre_launch:A,re:classloading,xf:fml:forge:itemstack,pl:mixin:APP:connectormod.mixins.json:item.ItemStackMixin from mod connectormod,pl:mixin:APP:betternether.mixins.common.json:ItemStackMixin from mod betternether,pl:mixin:APP:simple_quiver.mixins.json:ItemStackMixin from mod simple_quiver,pl:mixin:APP:cclayer.mixins.json:ItemStackMixin from mod curios,pl:mixin:APP:attributeslib.mixins.json:ItemStackMixin from mod attributeslib,pl:mixin:APP:spell_engine.mixins.json:ItemStackMixin from mod spell_engine,pl:mixin:APP:glitchcore.mixins.json:MixinItemStack from mod glitchcore,pl:mixin:APP:neruina.mixins.json:errorable.ItemStackMixin from mod neruina,pl:mixin:APP:bettercombat.mixins.json:ItemStackMixin from mod bettercombat,pl:mixin:APP:necronomicon-common.mixins.json:item.AnimatedItemNameMixin from mod necronomicon,pl:mixin:APP:fabric-item-api-v1.mixins.json:ItemStackMixin from mod fabric_item_api_v1,pl:mixin:APP:placebo.mixins.json:ItemStackMixin from mod placebo,pl:mixin:APP:accessories-tclayer.mixins.json:ItemStackMixin from mod tclayer,pl:mixin:APP:bclib.mixins.common.json:ItemStackMixin from mod bclib,pl:mixin:APP:mixins.deeperdarker.json:ItemStackMixin from mod deeperdarker,pl:mixin:APP:fastsuite.mixins.json:ItemStackMixin from mod fastsuite,pl:mixin:APP:armourersworkshop-mixins.json:CommonDataAttachMixin from mod armourers_workshop,pl:mixin:APP:connectormod.mixins.json:item.LateItemStackMixin from mod connectormod,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.server.level.ServerPlayerGameMode.m_6261_(ServerPlayerGameMode.java:286) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:computing_frames,pl:accesstransformer:B,pl:connector_pre_launch:A,re:mixin,pl:accesstransformer:B,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:spell_engine.mixins.json:action_impair.ServerPlayerInteractionManagerMixin from mod spell_engine,pl:mixin:APP:sanitydim.mixins.json:MixinServerPlayerGameMode from mod sanitydim,pl:mixin:APP:dimdoors-common.mixins.json:ServerPlayerInteractionManagerMixin from mod dimdoors,pl:mixin:APP:dimdoors.mixins.json:ServerPlayerGameModeMixin from mod dimdoors,pl:mixin:APP:fabric-events-interaction-v0.mixins.json:ServerPlayerInteractionManagerMixin from mod fabric_events_interaction_v0,pl:mixin:APP:item_obliterator.mixins.json:InteractionManagerMixin from mod item_obliterator,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.server.network.ServerGamePacketListenerImpl.m_5760_(ServerGamePacketListenerImpl.java:1085) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:mixin,pl:accesstransformer:B,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:fabric-networking-api-v1.mixins.json:ServerPlayNetworkHandlerMixin from mod fabric_networking_api_v1,pl:mixin:APP:modernfix-forge.mixins.json:bugfix.forge_vehicle_packets.ServerGamePacketListenerImplMixin from mod modernfix,pl:mixin:APP:forgivingvoid.mixins.json:ServerGamePacketListenerImplAccessor from mod forgivingvoid,pl:mixin:APP:krypton.mixins.json:shared.network.util.ServerPlayNetworkHandlerAccessor from mod krypton,pl:mixin:APP:badpackets.mixins.json:MixinServerGamePacketListenerImpl from mod badpackets,pl:mixin:APP:neruina.mixins.json:catchers.ServerPlayNetworkHandlerMixin from mod neruina,pl:mixin:APP:bettercombat.mixins.json:ServerPlayNetworkHandlerMixin from mod bettercombat,pl:mixin:APP:mixins/common/nochatreports.mixins.json:server.MixinServerGamePacketListenerImpl from mod nochatreports,pl:mixin:APP:adastra-common.mixins.json:common.ServerGamePacketListenerImplMixin from mod ad_astra,pl:mixin:APP:dimdoors-common.mixins.json:ExtendedServerPlayNetworkhandlerMixin from mod dimdoors,pl:mixin:APP:dimdoors-common.mixins.json:ServerPlayNetworkHandlerMixin from mod dimdoors,pl:mixin:APP:fabric-networking-api-v1.mixins.json:accessor.ServerPlayNetworkHandlerAccessor from mod fabric_networking_api_v1,pl:mixin:APP:tacz.mixins.json:common.ServerGamePacketListenerImplMixin from mod tacz,pl:mixin:APP:tacz.mixins.json:common.ServerPlayNetHandlerMixin from mod tacz,pl:mixin:APP:toofast.mixins.json:MixinServerNetPlayHandler from mod toofast,pl:mixin:APP:automobility.mixins.json:ServerGamePacketListenerImplMixin from mod automobility,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.network.protocol.game.ServerboundUseItemPacket.m_5797_(ServerboundUseItemPacket.java:29) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:classloading,pl:connector_pre_launch:A}
        at net.minecraft.network.protocol.game.ServerboundUseItemPacket.m_5797_(ServerboundUseItemPacket.java:7) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:classloading,pl:connector_pre_launch:A}
        at net.minecraft.network.protocol.PacketUtils.m_263899_(PacketUtils.java:22) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:classloading,pl:connector_pre_launch:A}
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:classloading,pl:connector_pre_launch:A}
        at net.minecraft.util.thread.BlockableEventLoop.m_6367_(BlockableEventLoop.java:156) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:computing_frames,pl:accesstransformer:B,pl:connector_pre_launch:A,re:mixin,pl:accesstransformer:B,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:modernfix-common.mixins.json:perf.fix_loop_spin_waiting.BlockableEventLoopMixin from mod modernfix,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.m_6367_(ReentrantBlockableEventLoop.java:23) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:computing_frames,pl:connector_pre_launch:A,re:mixin,pl:connector_pre_launch:A,re:classloading,pl:connector_pre_launch:A}
        at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:770) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:mixin:APP:witcher-class-mod.mixins.json:ExampleMixin from mod witcher_rpg,pl:mixin:APP:modernfix-common.mixins.json:core.MinecraftServerMixin from mod modernfix,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin from mod modernfix,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin from mod balm,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin from mod fastload,pl:mixin:APP:berserker-class-mod.mixins.json:ExampleMixin from mod berserker_rpg,pl:mixin:APP:neruina.mixins.json:MinecraftServerMixin from mod neruina,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer from mod crafttweaker,pl:mixin:APP:forcemaster-class-mod.mixins.json:ExampleMixin from mod forcemaster_rpg,pl:mixin:APP:fabric-message-api-v1.mixins.json:MinecraftServerMixin from mod fabric_message_api_v1,pl:mixin:APP:citadel.mixins.json:MinecraftServerMixin from mod citadel,pl:mixin:APP:elemental_wizards_rpg.mixins.json:ExampleMixin from mod elemental_wizards_rpg,pl:mixin:APP:connectormod.mixins.json:registries.MinecraftServerMixin from mod connectormod,pl:mixin:APP:fabric-lifecycle-events-v1.mixins.json:MinecraftServerMixin from mod fabric_lifecycle_events_v1,pl:mixin:APP:fabric-resource-loader-v0.mixins.json:MinecraftServerMixin from mod fabric_resource_loader_v0,pl:mixin:APP:archers-expansion.mixins.json:ExampleMixin from mod archers_expansion,pl:mixin:APP:additional_rpg_jewelry.mixins.json:ExampleMixin from mod additional_rpg_jewelry,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixin from mod bclib,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixinLate from mod bclib,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.server.MinecraftServer.m_6367_(MinecraftServer.java:161) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:mixin:APP:witcher-class-mod.mixins.json:ExampleMixin from mod witcher_rpg,pl:mixin:APP:modernfix-common.mixins.json:core.MinecraftServerMixin from mod modernfix,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin from mod modernfix,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin from mod balm,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin from mod fastload,pl:mixin:APP:berserker-class-mod.mixins.json:ExampleMixin from mod berserker_rpg,pl:mixin:APP:neruina.mixins.json:MinecraftServerMixin from mod neruina,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer from mod crafttweaker,pl:mixin:APP:forcemaster-class-mod.mixins.json:ExampleMixin from mod forcemaster_rpg,pl:mixin:APP:fabric-message-api-v1.mixins.json:MinecraftServerMixin from mod fabric_message_api_v1,pl:mixin:APP:citadel.mixins.json:MinecraftServerMixin from mod citadel,pl:mixin:APP:elemental_wizards_rpg.mixins.json:ExampleMixin from mod elemental_wizards_rpg,pl:mixin:APP:connectormod.mixins.json:registries.MinecraftServerMixin from mod connectormod,pl:mixin:APP:fabric-lifecycle-events-v1.mixins.json:MinecraftServerMixin from mod fabric_lifecycle_events_v1,pl:mixin:APP:fabric-resource-loader-v0.mixins.json:MinecraftServerMixin from mod fabric_resource_loader_v0,pl:mixin:APP:archers-expansion.mixins.json:ExampleMixin from mod archers_expansion,pl:mixin:APP:additional_rpg_jewelry.mixins.json:ExampleMixin from mod additional_rpg_jewelry,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixin from mod bclib,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixinLate from mod bclib,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.util.thread.BlockableEventLoop.m_7245_(BlockableEventLoop.java:130) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:computing_frames,pl:accesstransformer:B,pl:connector_pre_launch:A,re:mixin,pl:accesstransformer:B,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:modernfix-common.mixins.json:perf.fix_loop_spin_waiting.BlockableEventLoopMixin from mod modernfix,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.server.MinecraftServer.m_129961_(MinecraftServer.java:753) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:mixin:APP:witcher-class-mod.mixins.json:ExampleMixin from mod witcher_rpg,pl:mixin:APP:modernfix-common.mixins.json:core.MinecraftServerMixin from mod modernfix,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin from mod modernfix,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin from mod balm,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin from mod fastload,pl:mixin:APP:berserker-class-mod.mixins.json:ExampleMixin from mod berserker_rpg,pl:mixin:APP:neruina.mixins.json:MinecraftServerMixin from mod neruina,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer from mod crafttweaker,pl:mixin:APP:forcemaster-class-mod.mixins.json:ExampleMixin from mod forcemaster_rpg,pl:mixin:APP:fabric-message-api-v1.mixins.json:MinecraftServerMixin from mod fabric_message_api_v1,pl:mixin:APP:citadel.mixins.json:MinecraftServerMixin from mod citadel,pl:mixin:APP:elemental_wizards_rpg.mixins.json:ExampleMixin from mod elemental_wizards_rpg,pl:mixin:APP:connectormod.mixins.json:registries.MinecraftServerMixin from mod connectormod,pl:mixin:APP:fabric-lifecycle-events-v1.mixins.json:MinecraftServerMixin from mod fabric_lifecycle_events_v1,pl:mixin:APP:fabric-resource-loader-v0.mixins.json:MinecraftServerMixin from mod fabric_resource_loader_v0,pl:mixin:APP:archers-expansion.mixins.json:ExampleMixin from mod archers_expansion,pl:mixin:APP:additional_rpg_jewelry.mixins.json:ExampleMixin from mod additional_rpg_jewelry,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixin from mod bclib,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixinLate from mod bclib,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.server.MinecraftServer.m_7245_(MinecraftServer.java:747) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:mixin:APP:witcher-class-mod.mixins.json:ExampleMixin from mod witcher_rpg,pl:mixin:APP:modernfix-common.mixins.json:core.MinecraftServerMixin from mod modernfix,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin from mod modernfix,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin from mod balm,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin from mod fastload,pl:mixin:APP:berserker-class-mod.mixins.json:ExampleMixin from mod berserker_rpg,pl:mixin:APP:neruina.mixins.json:MinecraftServerMixin from mod neruina,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer from mod crafttweaker,pl:mixin:APP:forcemaster-class-mod.mixins.json:ExampleMixin from mod forcemaster_rpg,pl:mixin:APP:fabric-message-api-v1.mixins.json:MinecraftServerMixin from mod fabric_message_api_v1,pl:mixin:APP:citadel.mixins.json:MinecraftServerMixin from mod citadel,pl:mixin:APP:elemental_wizards_rpg.mixins.json:ExampleMixin from mod elemental_wizards_rpg,pl:mixin:APP:connectormod.mixins.json:registries.MinecraftServerMixin from mod connectormod,pl:mixin:APP:fabric-lifecycle-events-v1.mixins.json:MinecraftServerMixin from mod fabric_lifecycle_events_v1,pl:mixin:APP:fabric-resource-loader-v0.mixins.json:MinecraftServerMixin from mod fabric_resource_loader_v0,pl:mixin:APP:archers-expansion.mixins.json:ExampleMixin from mod archers_expansion,pl:mixin:APP:additional_rpg_jewelry.mixins.json:ExampleMixin from mod additional_rpg_jewelry,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixin from mod bclib,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixinLate from mod bclib,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.util.thread.BlockableEventLoop.m_18701_(BlockableEventLoop.java:139) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:computing_frames,pl:accesstransformer:B,pl:connector_pre_launch:A,re:mixin,pl:accesstransformer:B,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,pl:mixin:APP:modernfix-common.mixins.json:perf.fix_loop_spin_waiting.BlockableEventLoopMixin from mod modernfix,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.server.MinecraftServer.m_130012_(MinecraftServer.java:733) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:mixin:APP:witcher-class-mod.mixins.json:ExampleMixin from mod witcher_rpg,pl:mixin:APP:modernfix-common.mixins.json:core.MinecraftServerMixin from mod modernfix,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin from mod modernfix,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin from mod balm,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin from mod fastload,pl:mixin:APP:berserker-class-mod.mixins.json:ExampleMixin from mod berserker_rpg,pl:mixin:APP:neruina.mixins.json:MinecraftServerMixin from mod neruina,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer from mod crafttweaker,pl:mixin:APP:forcemaster-class-mod.mixins.json:ExampleMixin from mod forcemaster_rpg,pl:mixin:APP:fabric-message-api-v1.mixins.json:MinecraftServerMixin from mod fabric_message_api_v1,pl:mixin:APP:citadel.mixins.json:MinecraftServerMixin from mod citadel,pl:mixin:APP:elemental_wizards_rpg.mixins.json:ExampleMixin from mod elemental_wizards_rpg,pl:mixin:APP:connectormod.mixins.json:registries.MinecraftServerMixin from mod connectormod,pl:mixin:APP:fabric-lifecycle-events-v1.mixins.json:MinecraftServerMixin from mod fabric_lifecycle_events_v1,pl:mixin:APP:fabric-resource-loader-v0.mixins.json:MinecraftServerMixin from mod fabric_resource_loader_v0,pl:mixin:APP:archers-expansion.mixins.json:ExampleMixin from mod archers_expansion,pl:mixin:APP:additional_rpg_jewelry.mixins.json:ExampleMixin from mod additional_rpg_jewelry,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixin from mod bclib,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixinLate from mod bclib,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.server.MinecraftServer.m_130011_(MinecraftServer.java:665) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:mixin:APP:witcher-class-mod.mixins.json:ExampleMixin from mod witcher_rpg,pl:mixin:APP:modernfix-common.mixins.json:core.MinecraftServerMixin from mod modernfix,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin from mod modernfix,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin from mod balm,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin from mod fastload,pl:mixin:APP:berserker-class-mod.mixins.json:ExampleMixin from mod berserker_rpg,pl:mixin:APP:neruina.mixins.json:MinecraftServerMixin from mod neruina,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer from mod crafttweaker,pl:mixin:APP:forcemaster-class-mod.mixins.json:ExampleMixin from mod forcemaster_rpg,pl:mixin:APP:fabric-message-api-v1.mixins.json:MinecraftServerMixin from mod fabric_message_api_v1,pl:mixin:APP:citadel.mixins.json:MinecraftServerMixin from mod citadel,pl:mixin:APP:elemental_wizards_rpg.mixins.json:ExampleMixin from mod elemental_wizards_rpg,pl:mixin:APP:connectormod.mixins.json:registries.MinecraftServerMixin from mod connectormod,pl:mixin:APP:fabric-lifecycle-events-v1.mixins.json:MinecraftServerMixin from mod fabric_lifecycle_events_v1,pl:mixin:APP:fabric-resource-loader-v0.mixins.json:MinecraftServerMixin from mod fabric_resource_loader_v0,pl:mixin:APP:archers-expansion.mixins.json:ExampleMixin from mod archers_expansion,pl:mixin:APP:additional_rpg_jewelry.mixins.json:ExampleMixin from mod additional_rpg_jewelry,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixin from mod bclib,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixinLate from mod bclib,pl:mixin:A,pl:connector_pre_launch:A}
        at net.minecraft.server.MinecraftServer.m_206580_(MinecraftServer.java:251) ~[server-1.20.1-20230612.114412-srg.jar%23587!/:?] {re:mixin,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:connector_pre_launch:A,re:classloading,pl:accesstransformer:B,xf:fml:xaerominimap:xaero_minecraftserver,xf:fml:xaeroworldmap:xaero_wm_minecraftserver,pl:mixin:APP:witcher-class-mod.mixins.json:ExampleMixin from mod witcher_rpg,pl:mixin:APP:modernfix-common.mixins.json:core.MinecraftServerMixin from mod modernfix,pl:mixin:APP:modernfix-common.mixins.json:perf.dedicated_reload_executor.MinecraftServerMixin from mod modernfix,pl:mixin:APP:balm.mixins.json:MinecraftServerMixin from mod balm,pl:mixin:APP:fastload.mixins.json:server.MinecraftServerMixin from mod fastload,pl:mixin:APP:berserker-class-mod.mixins.json:ExampleMixin from mod berserker_rpg,pl:mixin:APP:neruina.mixins.json:MinecraftServerMixin from mod neruina,pl:mixin:APP:crafttweaker.mixins.json:common.access.server.AccessMinecraftServer from mod crafttweaker,pl:mixin:APP:forcemaster-class-mod.mixins.json:ExampleMixin from mod forcemaster_rpg,pl:mixin:APP:fabric-message-api-v1.mixins.json:MinecraftServerMixin from mod fabric_message_api_v1,pl:mixin:APP:citadel.mixins.json:MinecraftServerMixin from mod citadel,pl:mixin:APP:elemental_wizards_rpg.mixins.json:ExampleMixin from mod elemental_wizards_rpg,pl:mixin:APP:connectormod.mixins.json:registries.MinecraftServerMixin from mod connectormod,pl:mixin:APP:fabric-lifecycle-events-v1.mixins.json:MinecraftServerMixin from mod fabric_lifecycle_events_v1,pl:mixin:APP:fabric-resource-loader-v0.mixins.json:MinecraftServerMixin from mod fabric_resource_loader_v0,pl:mixin:APP:archers-expansion.mixins.json:ExampleMixin from mod archers_expansion,pl:mixin:APP:additional_rpg_jewelry.mixins.json:ExampleMixin from mod additional_rpg_jewelry,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixin from mod bclib,pl:mixin:APP:together.mixins.common.json:MinecraftServerMixinLate from mod bclib,pl:mixin:A,pl:connector_pre_launch:A}
        at java.lang.Thread.run(Thread.java:840) ~[?:?] {re:mixin}
```

**Comment by ArsenicPurple** [comment]
Should be fixed as of version 1.1.3 if you get the error again please post again with a full mod list
